src/main/resources/intentionDescriptions/ShareInPlaygroundIntention/description.html

```diff
@@ -1,5 +1,5 @@
 <html>
 <body>
-Shares code in Rust Playground (https://play.rust-lang.org/).
+Shares code in <a href="https://play.rust-lang.org/">Rust Playground</a>.
```

Comment:
👍

Reply:
I have noticed the same line in RsBundle. Should I also change it? Does html work in those localized strings?

Reply:
It depends on where the text is used. In the case of `action.Rust.ShareInPlayground.description` - it's used as a action description which doesn't support html so let's keep it as is

<img width="516" alt="Screenshot 2022-09-28 at 19 24 51" src="https://user-images.githubusercontent.com/2539310/192848870-62a157f9-66b2-424a-9f03-1e10772fd423.png">